src/main/java/org/opentripplanner/model/impl/TransferExportToCsvFile.java

```diff
@@ -0,0 +1,99 @@
+package org.opentripplanner.model.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import org.locationtech.jts.geom.Coordinate;
+import org.opentripplanner.common.geometry.SphericalDistanceLibrary;
+import org.opentripplanner.model.StopLocation;
+import org.opentripplanner.model.StopTime;
+import org.opentripplanner.model.TripStopTimes;
+import org.opentripplanner.model.transfer.StopTransferPoint;
+import org.opentripplanner.model.transfer.Transfer;
+import org.opentripplanner.model.transfer.TransferPoint;
+import org.opentripplanner.util.time.DurationUtils;
+import org.opentripplanner.util.time.TimeUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * This class can be used to export transfers for human verification. This is useful when trying
+ * to debug the rather complicated NeTEx data format.
+ */
+class TransferExportToCsvFile {
```

Comment:
This dump of the transfers with relational data proved to be very valuable for testing it, Andras suggested to make it available through the API. Maybe setting up a REST endpoint for various reports is a good idea.